<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modifierUnPret</class>
 <widget class="QDialog" name="modifierUnPret">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modifier un Prêt</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Numero du Prêt</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="numeroDuPrete">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>511</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Numero du Lecteur</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="numeroLecteur">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>511</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Numero du Livre</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="numeroLivre">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>511</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Date de Prêt</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="datePret">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>511</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Date de Retour</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateRetour">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>150</y>
     <width>511</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Recherche">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background:blue;
font-weight:bold;
color:white;
}</string>
   </property>
   <property name="text">
    <string>Rechercher</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Modifier">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>190</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background:green;
font-weight:bold;
color:white;
}</string>
   </property>
   <property name="text">
    <string>Modifier</string>
   </property>
  </widget>
  <widget class="QPushButton" name="annuler">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>190</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background:red;
font-weight:bold;
color:white;
}</string>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
